BusBoard.ConsoleApp/Program.cs

```diff
@@ -15,16 +17,34 @@ class Program
     static void Main(string[] args)
     {
       ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
-      var clientResponse = getClientJSONResponse();
+
+      var clientJSONResponse = getClientJSONResponse();
+      List<BusData> listArrivingBuses = getArrivingBussesFromJSON(clientJSONResponse);
+      printBusesToConsole(listArrivingBuses);
+    }
+
+    private static void printBusesToConsole(List<BusData> listArrivingBuses)
+    {
+      Console.WriteLine($"The next 5 buses arriving at {listArrivingBuses[0].StationName} are: \n");
+      foreach (var bus in listArrivingBuses)
+      {
+        Console.WriteLine(bus.VehicleID);
+      }
+    }
+
+
+    private static List<BusData> getArrivingBussesFromJSON(IRestResponse clientJSONResponse)
+    {
+      List<BusData> listArrivingBusDataFromJSON = JsonConvert.DeserializeObject<List<BusData>>(clientJSONResponse.Content);
+
+      return listArrivingBusDataFromJSON;
     }
 
     private static IRestResponse getClientJSONResponse()
     {
       var client = new RestClient("https://api.tfl.gov.uk");
       var request = new RestRequest("StopPoint/490008660N/Arrivals", DataFormat.Json);
       var response = client.Get(request);
```

Comment:
If you use `client.Execute<List<BusData>>(request).Data`, then it should get the data and parse it for you in one step. Which would mean that you can remove the constructor function in the `BusData` class and the `getArrivingBussesFromJSON` function.